### PR TITLE
Update Docker tutorial to remove Clear Containers reference 

### PR DESCRIPTION
**Pull request by puneetse**
Clear Containers runtime integration was removed. https://github.com/clearlinux-pkgs/docker/commit/a2b6e6f2eec9f34ea4abf5ee7deb28f378efc77c 

This PR is an update to remove the Clear Containers integration section accordingly and point to Kata Containers instead.

### CODE REVIEW

File: source/clear-linux/tutorials/docker/docker.rst [hunk 1]
```diff
@@ -68,48 +68,17 @@ packages to run Docker images as containers.
...
+`Kata Containers`_, is an open source project aiming to increase security
 of containers by using a hardware-backed virtual machine container runtime, 
-instead of software namespace containers that are provided by the standard 
+rather than software namespace containers which are provided by the standard 
 Docker *runc* runtime.
 
-The Docker package from |CL| will automatically use the *cc-runtime* 
-required for Clear Containers if it is available on your Clear Linux system. 
+Clear Linux provides easy integration of the *kata-runtime* with Docker.
+More information on installing and using  the *kata-runtime*  can be found at :ref:`kata`
```
[the remark this answers is not in the document]
Comment (puneetse) [reply source/clear-linux/tutorials/docker/docker.rst]: @mvincerx absolutely.

**Comment by mvincerx** [comment]
@puneetse. Will you please review my edits and advise?

**Comment by puneetse** [comment]
@mvincerx looks good, thank you.

**Comment by mvincerx** [comment]
@puneetse Thank you for your work on this!

**Pull request merged**